src/workers/parser/mapBindings.js

```diff
@@ -49,8 +79,10 @@ function replaceNode(ancestors, node) {
   }
 }
 
-function hasDestructuring(node) {
-  return node.declarations.some(declaration => t.isPattern(declaration.id));
+function hasNestedDestructuring(node) {
```

Comment:
Just a temporary change to make sure we don't process nested destructuring for now.